doc/kernel/other/interrupts.rst

```diff
@@ -57,6 +57,58 @@ nesting support is enabled.
     alter its behavior depending on whether it is executing as part of
     a thread or as part of an ISR.
 
+Multi-level Interrupt handling
+==============================
+
+A hardware platform can support more interrupt lines than natively-provided
+through the use of one or more nested interrupt controllers.  Sources of
+hardware interrupts are combined into one line that is then routed to
+the parent controller.
+
+If nested interrupt controllers are supported, option:`CONFIG_MULTI_LEVEL_INTERRUPTS`
```

Comment:
first colon is missing for the :option: command:
     
     :option:`CONFIG_MULTI_LEVEL_INTERRUPTS`